### PR TITLE
Add mounted event to table row

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
See changelog item

### CODE REVIEW

File: src/components/table/OcTable.vue [hunk 1]
```diff
@@ -445,10 +448,15 @@ export default {
           id: "9c4cf97e-5057-11eb-8044-b3d5df9caa21",
           resource: "this is fine.png",
           last_modified: 1599999999
-        }]
+        }];
+      }
+    },
+    methods: {
+      foo(a, b, c) {
```
Comment (kulmann) [inline src/components/table/OcTable.vue:455]: :trollface:

**Review by kulmann — COMMENTED** [review]
Why all the added semicolons? 🤔

File: src/components/table/OcTable.vue (hunk 1 above)
Comment (pascalwengerter) [reply src/components/table/OcTable.vue:455]: It's a feature? 🧛🏽

**Comment by pascalwengerter** [comment]
> Why all the added semicolons? 🤔

Fixed, I think Flo may have tinkered with the lint config during development or sth

**Review by kulmann — APPROVED** [review]
Love it! 💯

**Pull request merged**